src/osd/OSD.cc

```diff
@@ -4024,23 +4024,25 @@ void OSD::load_pgs()
       pg->upgrade(store);
     }
 
-    service.init_splits_between(pg->info.pgid, pg->get_osdmap(), osdmap);
```

Comment:
@dzafman David, do we need this code to generate PG's state for current mapping ?